packages/wonder-blocks-form/src/components/checkbox.tsx

```diff
@@ -6,9 +6,9 @@ import ChoiceInternal from "./choice-internal";
 // Keep synced with ChoiceComponentProps in ../util/types.js
 type ChoiceComponentProps = AriaProps & {
     /**
-     * Whether this component is checked
+     * Whether this component is checked or indeterminate
      */
-    checked: boolean;
+    checked: boolean | null | undefined;
```

Comment:
**suggestion:** we could use the `Checked` type exported from `checkbox-core.tsx` here.